clang/include/clang/AST/Stmt.h

```diff
@@ -1889,8 +1919,17 @@ class ValueStmt : public Stmt {
 
 private:
   WhereClause *WClause = nullptr;
+  BundlePositionBitfield PosInBundle;
 
 public:
+  /// Build a value statement.
+  ValueStmt(StmtClass SC) : Stmt(SC)
+  { PosInBundle.FirstStmt = 0; PosInBundle.LastStmt = 0; }
```

Comment:
The statements on this line and on line 1931 can also be written on different lines.